src/components/icon/icon.jsx

```diff
@@ -0,0 +1,25 @@
+'use strict';
+var React = require('react');
+
+module.exports = React.createClass({
+
+  propTypes: {
+    icon: React.PropTypes.string.isRequired,
+    iconFamily: React.PropTypes.oneOf(['font-awesome', 'glyphicon'])
+  },
+
+  render: function() {
+    var icon = this.props.icon;
+    var iconFamily = this.props.iconFamily || 'font-awesome';
```

Comment:
Rather than doing this, I'd set the default prop as font-awesome

```
getDefaultProps: function() {
  return {
    iconFamily: 'font-awesome'
  }
},
```